src/main/java/racingcar/InputView.java

```diff
@@ -6,15 +6,23 @@
  * 자동차 경주 Input
  */
 public class InputView {
-    Scanner sc = new Scanner(System.in);
+    private static final Scanner sc = new Scanner(System.in);
 
-    public int getNumberOfCars() {
-        System.out.println("자동차 대수는 몇 대 인가요?");
-        return Integer.parseInt(sc.nextLine());
+    public String[] insertCarName() {
+        System.out.println("경주할 자동차 이름을 입력하세요");
+        String text = sc.nextLine();
+        if(!text.contains(",")){
```

Comment:
```suggestion
        if (!text.contains(",")) {
```

IntelliJ Auto Formatting 기능을 사용하는 습관을 들여보아요.

The auto-formatting shortcut in IntelliJ

* For windows Ctrl + Alt + L.
* For ubuntu Ctrl + Alt + windows + L.
* For Mac Command + Option + L.